Gruntfile.js

```diff
@@ -1,35 +0,0 @@
-var cheerio = require('cheerio')
-
-module.exports = function (grunt) {
-  grunt.loadNpmTasks('grunt-link-checker');
-
-  grunt.initConfig({
-    pkg: grunt.file.readJSON('package.json'),
-
-    linkChecker: {
```

Comment:
All right, the link checker is back in place. It found two errors, one of which is [already fixed](https://github.com/electron/electron/commit/704433f3d620d5dcb49ad813848816615288372e) and another for which I have [opened a new PR](https://github.com/electron/electron/pull/8240).

That check is really handy. I would love to get it baked into the automated test suite somehow eventually.